internal/broadcaster/rate_broadcaster.go

```diff
@@ -182,6 +188,8 @@ utxoLoop:
 
 			b.satoshiMap.Store(tx.TxID(), tx.Outputs[0].Satoshis)
 
+			b.logger.Info("tx submitted", slog.String("tx id", tx.TxID()))
```

Comment:
This function does not submit txs, it just creates them. But I think we should remove this log as it will create super much logs if you do performance testing using this tool

Reply:
Removed